html/run3.html

```diff
@@ -30,6 +30,20 @@ <h5>Run 3 sample planning</h5>
             <td>{{row[2]}}</td>
           </tr>
         {% endfor %}
+	<!-- Space for free text inclusion -->  
+        <tr>
+          <td> <textarea style="width: 100%; margin-bottom: 8px;" placeholder="Add an identifier for the sample, e.g. name" id="datasetnamerun3">  </textarea> </td>
```

Comment:
Remove spaces from `<textarea>`. If you do `<textarea>  </textarea>` then not only your text value will have spaces at the beginning (if someone just clicks there and writes `"text"` you will get `"  text"`) but also placeholder is not showing up because now textarea is not empty.